### ISSUE TITLE
This package only works if you are using App\User 

### ISSUE THREAD

**Issue by inmanturbo** [venturedrake/laravel-crm#27]
- Laravel CRM Version: any
- PHP Version: 7.4 | 8.0

### Description:
This package only works if you are using App\User for auth

### Steps To Reproduce:

create  a user model anywhere other than App\User. Attempt to use package.

**Comment by andrewdrake** [comment]
Yes that is correct at the moment, in this pre-release stage :) We can work towards support for alternatives.

**Comment by andrewdrake** [comment]
@inmanturbo do you have an example for me where you gave User model in another location? Current the package will support App\User and App\Models\User.
